src/Pages/Order/AsideNav/ChoicePayment/ChoicePayment.js

```diff
@@ -57,44 +80,77 @@ export default class ChoicePayment extends Component {
+                <div className={detailBtnPushed ? "normal" : "hidden"}>
+                  {content}
+                </div>
               </div>
             );
           })}
         </div>
-        <div className="totalPrice">
-          <div>구매가격</div>
-          <div>₩107,990,000</div>
-        </div>
-        <div className="savingPrice">
-          <div>향후 5년 간 예상 연료비 절감</div>
-          <div>- ₩6,800,000</div>
-        </div>
-        <div className="afterSavingTotalPrice">
-          <div>예상 전체 절감 비용 반영 후 가격</div>
-          <div>₩101,190,000</div>
-        </div>
-        <div className="savingPriceDescription">
-          모든 절감 비용은 구매 이후 경험하게 됩니다.
+        <div
+          className={
+            detailBtnPushed
+              ? "detailPaymentWrap normal"
+              : "detailPaymentWrap hidden"
+          }
+        >
+          <div className="totalPrice">
+            <div>구매가격</div>
+            <div>{`₩${parseInt(total_price).toLocaleString()}`}</div>
+          </div>
+          <div className="savingPrice">
+            <div>향후 5년 간 예상 연료비 절감</div>
+            <div>- ₩6,800,000</div>
+          </div>
+          <div className="afterSavingTotalPrice">
+            <div>예상 전체 절감 비용 반영 후 가격</div>
+            <div>{`₩${parseInt(
+              fuel_cost_reduction_price
+            ).toLocaleString()}`}</div>
+          </div>
+          <div className="savingPriceDescription">
+            모든 절감 비용은 구매 이후 경험하게 됩니다.
+          </div>
+          <div className="savingPriceDescription">절감 계산 방법 보기</div>
         </div>
-        <div className="savingPriceDescription">절감 계산 방법 보기</div>
-        <button className="detailButton">
-          <div className="detailButtonDeco" />
-          세부 사항 감추기
+        <button
+          className="detailButton"
+          onClick={this.clickHandlerChangeDetailBtn}
+        >
+          <div className="detailButtonDeco">
+            <div
+              className={detailBtnPushed ? "detailBtn minus" : "detailBtn plus"}
+            />
+          </div>
+          {detailBtnPushed ? "세부 사항 감추기" : "세부 사항 표시"}
         </button>
         <div className="showTotalPrice">
           <ul>
-            <li>현금</li>
-            <li>리스</li>
-            <li>할부</li>
+            {payment.map((el, idx) => {
+              return (
+                <li
+                  className={
+                    activePaymentMethod === idx
+                      ? "paymentMethod active"
+                      : "paymentMethod normal"
+                  }
+                  key={idx}
+                  onClick={() => this.clickHandlerPaymentMethod(idx)}
+                >
+                  {el}
+                </li>
+              );
+            })}
           </ul>
           <div className="afterSavingTotalPrice">
             <div>예상 전체 절감 비용 반영 후 가격</div>
-            <div>₩101,190,000</div>
+            <div>{`₩${parseInt(
+              fuel_cost_reduction_price
+            ).toLocaleString()}`}</div>
```

Comment:
parseInt는 성능 문제가 있으므로 +연산자나 Number()를 사용해주세요
```suggestion
            <div>{`₩${+fuel_cost_reduction_price.toLocaleString()}`}</div>
```

Reply:
수정완료